src/App.js

```diff
@@ -1,9 +1,11 @@
 import "./assets/style/app.scss";
 import { Route, Routes } from "react-router-dom";
 import Home from "./pages/Home/Home";
```

Comment:
Can we add the full path of any component here ? 
```suggestion
import Home from "component/pages/Home/Home";
```